### PR TITLE
Osssio cidm neo profile 3.0.3 20170926 t163355 dev

### PR DESCRIPTION
**Pull request by eddo888**
https://servicesmadesimpler.govnet.qld.gov.au/jira/browse/QGCIDM-692

### CODE REVIEW

**Comment by ghost** [comment]
This looks fine.

**Comment by sachinkamdar** [comment]
This looks good...

**Comment by eddo888** [comment]
hi @william-dutton-dsiti could you please review this so I can push a test harness for Neo to DEV

File: OSSSIO/CIDM-NEO/profile/3.0.3_20170926T163355/DEV/js/cidm-test.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        checkUser();
+    }
+    else
+    {
+        alert("Please enter valid email address.");
+    }
+}
+
+function validateEmail(email) {
+  var re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (ThrawnCA) [inline OSSSIO/CIDM-NEO/profile/3.0.3_20170926T163355/DEV/js/cidm-test.js:72]: Where did this regex come from?

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**